lib/api/sentry_api.dart

```diff
@@ -32,14 +32,15 @@ class SentryApi {
     return _parseResponseList(response, (jsonMap) => Project.fromJson(jsonMap)).asFuture;
   }
 
-  Future<List<Release>> releases(String projectId, {int perPage = 25, int health = 1, int flatten = 0, String summaryStatsPeriod = '24h'}) async {
+  Future<List<Release>> releases(String organizationSlug, String projectId, {int perPage = 25, int health = 1, int flatten = 0, String summaryStatsPeriod = '24h'}) async {
```

Comment:
what's about named params for org and project Id but using `@required` for them? the idea is so that we don't need to care for the ordering of the params.

https://github.com/erluxman/awesomefluttertips/blob/master/page2.md#tip-27--dart-functionconstructor-arguments
https://pub.dev/documentation/meta/latest/meta/required-constant.html

Reply:
Yeah, i'm quite font of named parameters. 👍